Tests/StoreKitUnitTests/StoreKitConfigTestCase.swift

```diff
@@ -36,16 +37,20 @@ class StoreKitConfigTestCase: TestCase {
     var userDefaults: UserDefaults!
 
     override func setUpWithError() throws {
-        testSession = try SKTestSession(configurationFileNamed: "UnitTestsConfiguration")
-        testSession.resetToDefaultState()
-        testSession.disableDialogs = true
-        testSession.clearTransactions()
+        try super.setUpWithError()
+
+        try AvailabilityChecks.iOS14APIAvailableOrSkipTest()
```

Comment:
Just a question, can this still be called before iOS 14 with the `@available` annotation?

Reply:
It can. The annotation is only for compilation, but `XCTest` is not very smart and runs the tests anyway. That's why we have `AvailabilityChecks`.